frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "brace": "0.11.1",
     "connected-react-router": "4.5.0",
+    "diff-match-patch": "1.0.4",
```

Comment:
How do I add packages to frontend? Shouldn't yarn.lock also be updated?

We should document that:
https://mozilla-pontoon.readthedocs.io/en/latest/dev/contributing.html#dependencies

Reply:
It's a bit cumbersome but until I figure out how to deal with our docker volumes better, here's what I do:

```shell
cd ./frontend/
yarn add my-package
rm -rf node_modules
cd ..
make build
```

I'm making a note about the docs, I need to update a bunch of them anyway.